twingate/resource_resource.go

```diff
@@ -11,14 +11,33 @@ import (
 	"github.com/twingate/go-graphql-client"
 )
 
+const (
+	policyRestricted = "RESTRICTED"
+	policyAllowAll   = "ALLOW_ALL"
+	policyDenyAll    = "DENY_ALL"
+)
+
+func castToStrings(a, b interface{}) (string, string) {
+	return a.(string), b.(string)
+}
+
+func protocolDiff(k, oldValue, newValue string, d *schema.ResourceData) bool {
+	oldPolicy, newPolicy := castToStrings(d.GetChange("protocols.0.tcp.0.policy"))
```

Comment:
need to double-check, why here is only `tcp` protocol, seems like I didn't cover `udp` case

Reply:
fixed